docs/SCM-BestPractices/gitlab/README.md

```diff
@@ -17,14 +17,17 @@ This guide has been written for the:
 
 * **Maintainer** who wants to improve the security posture for one or more
 GitLab projects they support.
+* **Owner** who wants to improve the security posture for one or more GitLab
+groups they manage. 
 * **Open Source Program Office (OSPO)** who is typically responsible for
 multiple groups and projects.
 * **Operations** team tasked with applying policies as part of their work
 managing assets on GitLab.
+* **GitLab server admin** who wants to improve the security posture for their server. 
```

Comment:
need to find out what GitLab calls this.  is it server admin?  is enterprise admin?  something else?